lib/assessment/label.py

```diff
@@ -69,11 +70,14 @@ def cfe_label_student_work(self, rubric, student_code, code_feature_extractor, l
         for learning_goal in learning_goals:
             # Create instance of feature extractor
             cfe = CodeFeatures()
-            cfe.extract_features(student_code, learning_goal, lesson)
-            results["data"].append({"Label": cfe.assessment,
+            dt = DecisionTrees()
+            cfe.extract_features(student_code)
+            dt.assess(cfe.features, learning_goal, lesson)
+            results["data"].append({"Label": dt.assessment,
```

Comment:
Now that all code features are extracted for the code regardless of the learning goal and lesson, I think lines 72 and 74 can be moved above this loop to extract the features once for the code sample, then making a new DT assessment for each learning goal. 

I think this looks like:
```
        # Create instance of feature extractor
        cfe = CodeFeatures()
        cfe.extract_features(student_code)
        
        for learning_goal in learning_goals:
            dt = DecisionTrees()
            dt.assess(cfe.features, learning_goal, lesson)
            results["data"].append({"Label": dt.assessment,
```